src/lib.rs

```diff
@@ -1794,6 +1794,9 @@ mod api;
 mod binding;
 pub mod block;
 pub mod class;
+#[cfg(any(ruby_gte_3_0, docsrs))]
+#[cfg_attr(docsrs, doc(cfg(ruby_gte_3_0)))]
```

Comment:
Ruby 2.7 doesn't have `rb_obj_is_fiber`, and 2.7 support is nearing EOL anyway so adding workings doesn't seem worth. Not sure if I got the cfg right though...